src/main/java/leets/crazyform/global/error/ErrorCode.java

```diff
@@ -13,8 +13,10 @@ public enum ErrorCode {
     EXPIRED_TOKEN(403, "EXPIRED_TOKEN", "만료된 토큰입니다."),
     USER_NOT_FOUND(404, "USER_NOT_FOUND", "존재하지 않는 사용자입니다."),
     PASSWORD_NOT_MATCH(400, "PASSWORD_NOT_MATCH", "비밀번호가 일치하지 않습니다."),
-    EMAIL_DUPLICATE(409, "EMAIL_DUPLICATE", "중복된 이메일입니다.");
-
+    EMAIL_DUPLICATE(409, "EMAIL_DUPLICATE", "중복된 이메일입니다."),
+    WORKSPACE_NOT_FOUND(400, "WORKSPACE_NOT_FOUND", "워크스페이스를 찾을 수 없습니다."),
```

Comment:
```suggestion
    WORKSPACE_NOT_FOUND(404, "WORKSPACE_NOT_FOUND", "워크스페이스를 찾을 수 없습니다."),
```
NotFound는 `404` 에러코드를 사용합니다.